scripts/smoke_test.sh

```diff
@@ -4,8 +4,8 @@ set -ex
 
 # provide a default environment if none exists
 if [ ! -f ".env" ]; then
-    echo "Using .env.oss"
-    cp .env.oss .env
+    echo "Using .env.example and .env.oss as .env"
+    cat .env.example > .env && cat .env.oss >> .env
```

Comment:
```suggestion
    cat .env.example .env.oss > .env
```

Reply:
going to undo this change as `.env.example` has been reduced to app settings and a couple of local vars.